app/Http/Controllers/Profile/ProfileController.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Controllers\Profile;
+
+use Kris\LaravelFormBuilder\FormBuilderTrait;
+use Illuminate\Http\Request;
+use App\Member;
+use App\Forms\UpdateProfileForm;
+use App\Http\Requests;
+use App\Http\Controllers\Controller;
+use Auth;
+use JsValidator;
+use Session;
+
+class ProfileController extends Controller
+{
+    use FormBuilderTrait;
+	
+    public function show()
+    {
+		$user = Auth::user();
+		
+		if($user === null){
```

Comment:
The canonical (and easier!) way to do login checks is not in the controller itself, but in routing using middleware. Lines 21-27 should be removed.

Making sure that the user is logged in should be done via requiring the middleware `auth`, which can be done by adding one line to the constructor - see [HomeController.php#L17](https://github.com/CUMSA/members/blob/9f43d079835d7c72dc268a9608f87d08e3ad752f/app/Http/Controllers/HomeController.php#L17) for an example of this being done.